io.github.appoutlet.GameOutlet.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.appoutlet.GameOutlet
+runtime: org.gnome.Platform
```

Comment:
do you really need the GNOME platform?

Reply:
I think so. I'm using `Desktop.getDesktop().browse(URI(url))` to open links in browser. Under the hood, this line checks the supported actions. I found this internal class
 
![Screenshot from 2023-06-18 15-17-10](https://github.com/flathub/flathub/assets/10220064/18f23b90-a317-482b-a657-21c84a1d7cfa)

When I use `org.freedesktop.Sdk` I've got "The BROWSE action is not supported in your platform"